src/pages/guides/reference/configuringproxy.md

```diff
@@ -0,0 +1,15 @@
+# Configuring Proxy 
+
+Runtime does not expose egress IPs due to security reasons. If customers need a way to securely communicate with their backend services, they can use a proxy in between their system and Runtime. 
+
+This can be done as following: 
+
+1. Setting up an Azure Front Door
+2. Configuring a CDN in front of Runtime 
```

Comment:
Are we going to provide a tutorial for CDN configuration specific for IO Runtime?

Reply:
We can add it later. For now, we can star with options

Reply:
Move Cosmin's PR - Configuring CDN here.